tf-modules/archive/main.tf

```diff
@@ -3,3 +3,15 @@ terraform {
     aws = ">= 2.31.0"
   }
 }
+
+locals {
+  subnet_ids = var.lambda_subnet_ids == null ? [] : var.lambda_subnet_ids
```

Comment:
This variable was called `lambda_subnet_ids` instead of `subnet_ids` so that we would have the ability to specify one set of subnets for Lambda functions and another set for other things (like EC2 instances or ES domains). I'd rather see that kept instead of the more generic `local.subnet_ids`.

Reply:
Ah, sure thing. I'll update.